### PR TITLE
Feature/shopify multipass integration

### PR DESCRIPTION
**Pull request by clairekinde11**
A new topic describing how to integrate Shopify Multipass with Kinde.

## Summary by CodeRabbit

- **New Features**
	- Introduced documentation for integrating Kinde with Shopify's Multipass feature, enhancing user authentication between Kinde and Shopify.
	- Updated sidebar navigation order for improved accessibility of Kinde integration documents with Cloudflare Zero Trust and WordPress.

- **Bug Fixes**
	- Adjusted sidebar order for Kinde and WordPress integration documentation to improve visibility and navigation.

### CODE REVIEW

File: src/content/docs/integrate/third-party-tools/shopify-multipass.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      .withDomain(process.env.SHOPIFY_STORE_URL as string)
+      .url();
+
+    return NextResponse.redirect(url, 302)
+  } else {
+    const params = new URLSearchParams();
+    params.append('post_login_redirect_url', `${process.env.KINDE_SITE_URL}/api/multipass`);
+    return NextResponse.redirect(`${process.env.KINDE_SITE_URL}/api/auth/login?${params.toString()}`, 302);
+  }
+}
```
[the remark this answers is not in the document]
Comment (clairekinde11) [reply src/content/docs/integrate/third-party-tools/shopify-multipass.mdx:68]: @DanielRivers is coderabbit right?

[the remark this answers is not in the document]
Comment (DanielRivers) [reply src/content/docs/integrate/third-party-tools/shopify-multipass.mdx:68]: @clairekinde11 Yes! it is

[the remark this answers is not in the document]
Comment (DanielRivers) [reply src/content/docs/integrate/third-party-tools/shopify-multipass.mdx]: Fixed it

**Review by marcosmartini — APPROVED** [review]
LGTM

**Pull request merged**